config/config.go

```diff
@@ -775,8 +775,8 @@ func (c *Config) ToOptions() skipper.Options {
 			Append:  c.AppendFilters.filters,
 		},
 		DisabledFilters:    c.DisabledFilters.values,
-		CloneRoute:         eskip.NewClone(c.CloneRoute.Reg, c.CloneRoute.Repl),
-		EditRoute:          eskip.NewEditor(c.EditRoute.Reg, c.EditRoute.Repl),
+		CloneRoute:         []*eskip.Clone{},
```

Comment:
No need to init with empty slice as `append` works on nil slices https://go.dev/tour/moretypes/15

Reply:
https://www.youtube.com/watch?v=ynoY2xz-F8s is nice to watch to understand nil in Go. It's fundamental different than in other languages. You can even do something like this to work around nil ptr panic:

```
type t struct{
    s string
}

func (t *t) String() string {
    if t == nil {
        t := new(t)
    }
    return t.s
}
```